db/migrate/20210627100932_create_assignments.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class CreateAssignments < ActiveRecord::Migration[6.1]
+  def change
+    create_table :assignments do |t|
+      t.integer :topic_id, null: false
+      t.integer :assigned_to_id, null: false
```

Comment:
How are you planning on using this to differentiate between `users` and `groups`?

Reply:
A polymorphic `has_one` association, using a (to-be-added) `assigned_to_type` column.